hibernate-community-dialects/src/main/java/org/hibernate/community/dialect/DB2LegacySqlAstTranslator.java

```diff
@@ -333,15 +332,6 @@ protected void visitUpdateStatementOnly(UpdateStatement statement) {
 		}
 	}
 
-	@Override
-	protected void visitInsertStatementOnly(InsertStatement statement) {
-		final boolean closeWrapper = renderReturningClause( statement );
-		super.visitInsertStatementOnly( statement );
-		if ( closeWrapper ) {
-			appendSql( ')' );
-		}
-	}
```

Comment:
I guess the removal of this was a victim of the renaming of `InsertStatement`? This should be aligned with `DB2SqlAstTranslator`.